linux/keyman-config/keyman_config/install_kmp.py

```diff
@@ -65,143 +69,218 @@ def get_infdata(tmpdirname):
 	else:
 		return None, None, None, None, None
 
-def install_kmp(inputfile, online=False):
+def download_source(kbid, kbdir, sourcePath):
+	# just get latest version of kmn unless there turns out to be a way to get the version of a file at a date
+	base_url = "https://raw.github.com/keymanapp/keyboards/master/" + sourcePath
+	kmn_url = base_url + "/source/" + kbid + ".kmn"
+	response = requests.get(kmn_url)
+	if response.status_code == 200:
+		kmndownloadfile = os.path.join(kbdir, kbid + ".kmn")
+		with open(kmndownloadfile, 'wb') as f:
+			f.write(response.content)
+			logging.info("Installing %s.kmn as keyman file", kbid)
+		kmn_file = os.path.join(kbdir, kbid + ".kmn")
```

Comment:
Isn't this also
`kmn_file = kmndownloadfile`?

Reply:
good point, done in next commit